src/com/projectkorra/projectkorra/board/BendingBoardManager.java

```diff
@@ -45,15 +50,27 @@ public static void reload() {
 	}
 
 	private static void initialize(boolean initial) {
-		if (initial) loadDisabledPlayers();
+		if (initial) {
+			loadDisabledPlayers();
```

Comment:
Given my understanding of what this function does, I have a few complaints with it as a whole, not necessarily your changes in specific since it was purely syntactic.

For readability and ease of comprehension, I feel that this should be split into 2 methods, the initialize method should only be called once and shouldn't require a boolean value. We should create a separate load method for all of these, or analyze why we only load disabled players in the initial case.